### PR TITLE
Fix/longer session identifiers

### PR DESCRIPTION
**Pull request by marcoconti83**
# Reason for this pull request
Adopt the new APIs of data model, mock transport and cryptobox

### CODE REVIEW

File: Tests/Source/E2EE/UserClientRequestFactoryTests.swift [hunk 1]
```diff
@@ -28,25 +28,24 @@ class UserClientRequestFactoryTests: MessagingTest {
     
     var sut: UserClientRequestFactory!
     var authenticationStatus: ZMAuthenticationStatus!
+    var spyKeyStore: SpyUserClientKeyStore!
```
Comment (typfel) [inline Tests/Source/E2EE/UserClientRequestFactoryTests.swift]: :)

**Comment by typfel** [comment]
The project file changes are necessary?

File: Tests/Source/E2EE/UserClientRequestFactoryTests.swift (hunk 1 above)
Comment (marcoconti83) [inline Tests/Source/E2EE/UserClientRequestFactoryTests.swift]: It's actually a technical term :)

Comment (marcoconti83) [inline Tests/Source/E2EE/UserClientRequestFactoryTests.swift]: for example, see: http://stackoverflow.com/a/8758060/412927

**Pull request merged**